### ISSUE TITLE
QueryFilter.FilterOperator.Equals() with null value

### ISSUE THREAD

**Issue by dietert** [keenlabs/keen-sdk-net#23]
Hi,

I want to check if a property is null.

I'll do this with the following code

```
new QueryFilter("propertyname", QueryFilter.FilterOperator.Equals(), null)
```

But it throws an ArgumentNullException

Should I build my filter different? Or is my code correct and shouldn't the library throwing the ArgumentNullException?

Thanks for your answer!

Regards,
Dieter

**Comment by masojus** [comment]
Just taking a look at the behavior here, and since the API does allow for a given property to be `null` and pass an `exists` filter with a value of `true`, and of course a property can have the string value `'null'` there is a meaningful distinction here that needs to be addressed. The Java SDK has this same bug, I believe.

**Comment by masojus** [comment]
Fixed in commit 60a31f3
